components/feature/intent/src/main/java/mozilla/components/feature/intent/processing/TabIntentProcessor.kt

```diff
@@ -83,16 +86,17 @@ class TabIntentProcessor(
         return if (searchQuery.isNullOrBlank()) {
             false
         } else {
+            val source = SessionState.Source.External.ActionView(intent.externalPackage())
```

Comment:
This should be `External.ActionSearch`. Can we cover this in TabIntentProcessorTest?